app/views/items/edit.html.erb

```diff
@@ -1,20 +1,22 @@
-<% content_for(:title, "Edit Item") %>
-<h1>Edit Item</h1>
-<%= simple_form_for @item, url: item_path(@item), method: :patch do |f| %>
-  <%= f.error_notification %>
+<div class="container align-justify-center">
+  <% content_for(:title, "Edit Item") %>
+  <h1>Edit Item</h1>
+  <%= simple_form_for @item, url: item_path do |f| %>
+    <%= f.error_notification %>
 
-  <%= f.input :title, label: 'Iphone:' %>
+    <%= f.input :title, placeholder: 'Macbook', label:'Give a sexy Title:' %>
 
-  <%= f.input :description, label: 'Useful for Students:' %>
+    <%= f.input :description, placeholder: 'Great for student', label:'Description:' %>
 
-  <%= f.input :condition, label: 'Excellent:' %>
+    <%= f.input :condition, placeholder: 'Excellent', label:'Condition = Excellent, Good, Average, Bad, Broken:' %>
```

Comment:
Suggest to change the label to `label: 'Condition'`, then add in `, collection: %w[Excellent Good Average Bad Broken]` . This will give the user a dropdown selection.

Reply:
Added in Collection to condition